src/main/java/frc/robot/commands/BounceAuto.java

```diff
@@ -0,0 +1,159 @@
+package frc.robot.commands;
+
+import java.util.List;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryConfig;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryGenerator;
+import edu.wpi.first.wpilibj.trajectory.constraint.DifferentialDriveVoltageConstraint;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.FieldMapHome;
+import frc.robot.subsystems.DriveTrain;
+public class BounceAuto extends SequentialCommandGroup implements AutoCommandInterface {
+  Rotation2d rotation180 = Rotation2d.fromDegrees(180.0);
+  Rotation2d rotation270 = Rotation2d.fromDegrees(270.0);
+  Rotation2d rotation90 = Rotation2d.fromDegrees(90.0);
```

Comment:
these 3 should probably be "final".

Reply:
Also, the indent of these lines is wrong.

Reply:
I fixed those changes.